.github/workflows/workflow.yml

```diff
@@ -0,0 +1,14 @@
+name: check-terraform
+run-name: ${{ github.actor }} triggered
+on: [push, pull_request]
+jobs:
+  check_format:
+      runs-on: ubuntu-latest
```

Comment:
this workflow mentions it needs to check that runs on ubuntu latest, We run on OpenCloudOS. What does this refer to exactly?

Reply:
the workflow only check and validate the code styles, can be run on any OS.

Reply:
ok, good